mmaction/core/evaluation/accuracy.py

```diff
@@ -149,6 +149,7 @@ def mean_average_precision(scores, labels):
             scores[i], labels[i])
         ap = -np.sum(np.diff(recall) * np.array(precision)[:-1])
         results.append(ap)
+    results = [x for x in results if not np.isnan(x)]
```

Comment:
curios, can results be empty?

Reply:
It happens when a class exists in the training set but doesn't exist in the testing set. (A bug in the released HVU dataset).

Reply:
It happens when a class exists in the training set but doesn't exist in the testing set. [A quick fix of a bug I've written, for the HVU object class, I wrongly set the num_classes to 1679 (actually it is 1678)].  I'll fix it in the next version.

Reply:
mean of empty is not defined. try np.mean([]) to see the warning